### PR TITLE
add in free-courses-hi frameworks like spring boot,spring core,spring…

### PR DESCRIPTION
**Pull request by Himanshusinghtomar**
… mvc,Hibernate,servlet jsp , hosting on aws java

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Himanshusinghtomar** [comment]
Provided few more framework courses on java

**Comment by Himanshusinghtomar** [comment]
include all inside java index

**Comment by Himanshusinghtomar** [comment]
add java hindi courses

**Comment by Himanshusinghtomar** [comment]
Resolved Conflict

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -300,8 +300,9 @@
 
 ### Java
 
-* [Complete Java Programming in Hindi](https://www.youtube.com/playlist?list=PLmRclvVt5DtnqhXTJwd-oqVRwO3bLZCGV) - Anand Kumar, CodeitUp
+* [AWS Hosting Tutorial | Deploying Java Applications on Amazon EC2 in Hindi](https://youtube.com/playlist?list=PL0zysOflRCek6D-kLRM48rhyvTEbc60nc&si=9ILVhPNVuvemweEJ) - Learn Code With Durgesh
```
Comment (eshellman) [inline courses/free-courses-hi.md:303]: escape the `|`

**Comment by eshellman** [comment]
when you've made changes, please add `(changes made)` to the title of your PR

File: courses/free-courses-hi.md (hunk 1 above)
Comment (eshellman) [reply courses/free-courses-hi.md:304]:
```suggestion
* [AWS Hosting Tutorial \| Deploying Java Applications on Amazon EC2 in Hindi](https://youtube.com/playlist?list=PL0zysOflRCek6D-kLRM48rhyvTEbc60nc) - Learn Code With Durgesh
```

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -301,18 +301,23 @@
...
 * [Core Java Programming (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhij8Oplrvjt_RlDliZQgdxoV) - Rajesh Kumar, Geeky Shows
+* [Hibernate tutorial Basic to Advance in Hindi](https://youtube.com/playlist?list=PL0zysOflRCekX8OO7V7pGQ9kxZ28JyJlk&si=thD-RnaLu6VPDpDr) - Learn Code With Durgesh
```
Comment (eshellman) [inline courses/free-courses-hi.md:306]:
```suggestion
* [Hibernate tutorial Basic to Advance in Hindi](https://youtube.com/playlist?list=PL0zysOflRCekX8OO7V7pGQ9kxZ28JyJlk) - Learn Code With Durgesh
```

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -301,18 +301,23 @@
...
-* [Java Foundation Course \| Hindi](https://www.youtube.com/playlist?list=PL-Jc9J83PIiFj7YSPl2ulcpwy-mwj1SSk) - Pepcoding
+* [Java Foundation Course Hindi](https://www.youtube.com/playlist?list=PL-Jc9J83PIiFj7YSPl2ulcpwy-mwj1SSk) - Pepcoding
 * [Java Live Community Class](https://www.youtube.com/playlist?list=PLsyeobzWxl7pyuXTES7ZqncOI_X0mgKgK) - Navin Reddy (Telusko)
 * [Java Programming Tutorial (HINDI/URDU)](https://www.youtube.com/playlist?list=PLiOa6ike4WAHljIOitb3vR0nXQgneUedR) - Vikas Pandey, Easytuts4you
 * [Java Tutorial](https://youtube.com/playlist?list=PLX9Zi6XTqOKQ7TdRz0QynGIKuMV9Q2H8E) - Saurabh Shukla Sir
 * [Java Tutorials for Beginners](https://www.youtube.com/playlist?list=PLlhM4lkb2sEhfuXL-2BDrJ67WkUdQ2v9b) - Deepak Panwar, Smart Programming
 * [Java Tutorials For Beginners In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9agS67Uits0UnJyrYiXhDS6q) - CodeWithHarry
+* [Servlet and Jsp](https://youtube.com/playlist?list=PL0zysOflRCel5BSXoslpfDawe8FyyOSZb&si=pyUQrpB8bp-pbrR7) - Learn Code With Durgesh
+* [Spring Boot tutorial basic in Hindi](https://youtube.com/playlist?list=PL0zysOflRCelmjxj-g4jLr3WKraSU_e8q&si=nZizMDReuxWBM3hB) - Learn Code With Durgesh
```
Comment (eshellman) [inline courses/free-courses-hi.md:317]:
```suggestion
* [Spring Boot tutorial basic in Hindi](https://youtube.com/playlist?list=PL0zysOflRCelmjxj-g4jLr3WKraSU_e8q) - Learn Code With Durgesh
```

File: courses/free-courses-hi.md [hunk 4]
```diff
@@ -301,18 +301,23 @@
...
 * [Java and DSA Foundation Course](https://youtube.com/playlist?list=PLxgZQoSe9cg00xyG5gzb5BMkOClkch7Gr&si=ctLRknqelKTSI5Lh) - College Wallah
-* [Java Foundation Course \| Hindi](https://www.youtube.com/playlist?list=PL-Jc9J83PIiFj7YSPl2ulcpwy-mwj1SSk) - Pepcoding
+* [Java Foundation Course Hindi](https://www.youtube.com/playlist?list=PL-Jc9J83PIiFj7YSPl2ulcpwy-mwj1SSk) - Pepcoding
 * [Java Live Community Class](https://www.youtube.com/playlist?list=PLsyeobzWxl7pyuXTES7ZqncOI_X0mgKgK) - Navin Reddy (Telusko)
 * [Java Programming Tutorial (HINDI/URDU)](https://www.youtube.com/playlist?list=PLiOa6ike4WAHljIOitb3vR0nXQgneUedR) - Vikas Pandey, Easytuts4you
 * [Java Tutorial](https://youtube.com/playlist?list=PLX9Zi6XTqOKQ7TdRz0QynGIKuMV9Q2H8E) - Saurabh Shukla Sir
 * [Java Tutorials for Beginners](https://www.youtube.com/playlist?list=PLlhM4lkb2sEhfuXL-2BDrJ67WkUdQ2v9b) - Deepak Panwar, Smart Programming
 * [Java Tutorials For Beginners In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9agS67Uits0UnJyrYiXhDS6q) - CodeWithHarry
+* [Servlet and Jsp](https://youtube.com/playlist?list=PL0zysOflRCel5BSXoslpfDawe8FyyOSZb&si=pyUQrpB8bp-pbrR7) - Learn Code With Durgesh
```
Comment (eshellman) [inline courses/free-courses-hi.md:316]:
```suggestion
* [Servlet and Jsp](https://youtube.com/playlist?list=PL0zysOflRCel5BSXoslpfDawe8FyyOSZb) - Learn Code With Durgesh
```

**Review by eshellman — CHANGES_REQUESTED** [review]
also remove tracking code for last to spring tutorials.
when you've made changes, please add `(changes made)` to the title of your PR

**Review by Himanshusinghtomar — COMMENTED** [review]
change made

**Comment by Himanshusinghtomar** [comment]
all change made

File: courses/free-courses-hi.md (hunk 2 above)
Comment (Himanshusinghtomar) [reply courses/free-courses-hi.md]: change made

File: courses/free-courses-hi.md [hunk 5]
```diff
@@ -302,19 +302,22 @@
 
 ### Java
 
-* [Complete Java Programming in Hindi](https://www.youtube.com/playlist?list=PLmRclvVt5DtnqhXTJwd-oqVRwO3bLZCGV) - Anand Kumar, CodeitUp
+* [AWS Hosting Tutorial | Deploying Java Applications on Amazon EC2 in Hindi](https://youtube.com/playlist?list=PL0zysOflRCek6D-kLRM48rhyvTEbc60nc) - Learn Code With Durgesh
 * [Core Java Programming (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhij8Oplrvjt_RlDliZQgdxoV) - Rajesh Kumar, Geeky Shows
+* [Hibernate tutorial Basic to Advance in Hindi](https://youtube.com/playlist?list=PL0zysOflRCekX8OO7V7pGQ9kxZ28JyJlk) - Learn Code With Durgesh
 * [Java + DS + Algorithms](https://www.youtube.com/playlist?list=PLKKfKV1b9e8ps6dD3QA5KFfHdiWj9cB1s) - Apni Kaksha (Anuj)
 * [Java + DSA](https://www.youtube.com/playlist?list=PLfqMhTWNBTe3LtFWcvwpqTkUSlB32kJop) - Apna College
-* [Java and DSA Foundation Course](https://youtube.com/playlist?list=PLxgZQoSe9cg00xyG5gzb5BMkOClkch7Gr&si=ctLRknqelKTSI5Lh) - College Wallah
```
Comment (eshellman) [inline courses/free-courses-hi.md:309]: did you mean to remove this?

File: courses/free-courses-hi.md [hunk 6]
```diff
@@ -302,19 +302,22 @@
...
-* [Java Foundation Course \| Hindi](https://www.youtube.com/playlist?list=PL-Jc9J83PIiFj7YSPl2ulcpwy-mwj1SSk) - Pepcoding
+* [Java Foundation Course | Hindi](https://www.youtube.com/playlist?list=PL-Jc9J83PIiFj7YSPl2ulcpwy-mwj1SSk) - Pepcoding
```
Comment (eshellman) [inline courses/free-courses-hi.md:310]: did you mean to revert this

**Review by eshellman — CHANGES_REQUESTED** [review]
please be careful rsolving conflicts

File: courses/free-courses-hi.md (hunk 6 above)
Comment (Himanshusinghtomar) [reply courses/free-courses-hi.md:310]: yes

**Comment by Himanshusinghtomar** [comment]
change done

**Comment by eshellman** [comment]
It looks to me like you inadvertently removed good resources while trying to resolve conflicts. when resolving conflicts, please do not undo the work that others have done.

**Comment by eshellman** [comment]
course added in fe15c9c738c82b1d7c032585304ef63c7b6fa15e

**Pull request closed without merging**